config/packages/framework.yaml

```diff
@@ -0,0 +1,51 @@
+framework:
+    assets:
+        json_manifest_path: '%kernel.project_dir%/public/build/manifest.json'
+
+    cache:
+        # this value is used as part of the "namespace" generated for the cache item keys
+        # to avoid collisions when multiple apps share the same cache backend (e.g. a Redis server)
+        # See https://symfony.com/doc/current/reference/configuration/framework.html#prefix-seed
+        prefix_seed: symfony-demo
+
+    csrf_protection: null
+
+    default_locale: '%locale%'
+
+    # When using the HTTP Cache, ESI allows to render page fragments separately
+    # and with different cache configurations for each fragment
+    # https://symfony.com/doc/current/book/http_cache.html#edge-side-includes
+    esi: { enabled: true }
+
+    form: null
+
+    fragments: null
+
+    http_method_override: true
+
+    # The 'ide' option turns all of the file paths in an exception page
+    # into clickable links that open the given file using your favorite IDE.
+    # When 'ide' is set to null the file is opened in your web browser.
+    # See https://symfony.com/doc/current/reference/configuration/framework.html#ide
+    ide: ~
+
+    php_errors:
+        log: true
+
+    secret: '%env(APP_SECRET)%'
+
+    session:
+        handler_id: session.handler.native_file
+        save_path: '%kernel.project_dir%/var/sessions/%kernel.environment%'
+
+    trusted_hosts: null
```

Comment:
I think it's `trusted_proxies` the one that's deprecated: https://symfony.com/blog/fixing-the-trusted-proxies-configuration-for-symfony-3-3